### PR TITLE
Make times in drop down respect BST

### PR DESCRIPTION
**Pull request by philwills**

### CODE REVIEW

File: riff-raff/app/controllers/DeployController.scala [hunk 1]
```diff
@@ -149,12 +149,13 @@ object DeployController extends Controller with Logging with LoginActions {
...
 
+  val shortFormat = DateTimeFormat.forPattern("HH:mm d/M/yy").withZone(DateTimeZone.forID("Europe/London"))
+
   def autoCompleteBuild(project: String, term: String) = AuthAction {
     val possibleProjects = Builds.successfulBuilds(project).filter(
       p => p.number.contains(term) || p.branchName.contains(term)
     ).map { build =>
-      val formatter = DateTimeFormat.forPattern("HH:mm d/M/yy")
-      val label = "%s [%s] (%s)" format(build.number, build.branchName, formatter.print(build.startTime))
+      val label = "%s [%s] (%s)" format(build.number, build.branchName, shortFormat.print(build.startTime))
```
Comment (adamnfish) [inline riff-raff/app/controllers/DeployController.scala]: Why use format over string interpolation?

**Comment by adamnfish** [comment]
:+1: Excellent

File: riff-raff/app/controllers/DeployController.scala (hunk 1 above)
Comment (philwills) [inline riff-raff/app/controllers/DeployController.scala]: Good question. I suspect just because this is quite old code.

**Comment by sihil** [comment]
Ideally this should respect the user's timezone using the locale in the request (times rendered locally using moment do this). I had a brief look at this, but it looked non-trivial with the way the templates were arranged.

**Comment by philwills** [comment]
@sihil That does sound ideal, but I don't want the best to be the enemy of the good

**Pull request merged**